pkgs/by-name/pi/pipeline/package.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  appstream,
+  blueprint-compiler,
+  cairo,
+  cargo,
+  darwin,
+  desktop-file-utils,
+  gdk-pixbuf,
+  gettext,
+  glib,
+  gtk4,
+  libadwaita,
+  meson,
+  ninja,
+  openssl,
+  pango,
+  pkg-config,
+  rustc,
+  rustPlatform,
+  wrapGAppsHook4,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pipeline";
+  version = "1.15.0";
+
+  src = fetchFromGitLab {
+    owner = "schmiddi-on-mobile";
+    repo = "pipeline";
+    rev = "v${version}";
+    hash = "sha256-tZyAQz7mhd+YXaO6+XYpUxza5ViVELE3J0Zeu11fr/U=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "piped-openapi-sdk-1.0.0" = "sha256-UFzMYYqCzO6KyJvjvK/hBJtz3FOuSC2gWjKp72WFEGk=";
+      "tf_core-0.1.4" = "sha256-IW5d0mn/olgm9ydN45ZaDd5AQSGj2kM7QvCHgZSnd8w=";
+    };
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook4
+    rustPlatform.cargoSetupHook
+    meson
+    gettext
+    glib
+    pkg-config
+    desktop-file-utils
+    appstream
+    ninja
+    rustc
+    cargo
+    openssl
```

Comment:
Why do we need openssl in nativeBuildInputs?

Reply:
That doesn't answer my question. Their flake can also be wrong which I think it is. We usually don't want to use openssl-config but pkg-config and besides the openssl cli I don't think there are other executables in the package.

I tried building the package locally and openssl was not required here.

Reply:
```suggestion
```